### ISSUE TITLE
Having insight monitoring issues again, "no data available"

### ISSUE THREAD

**Issue by jaxjexjox** [opnsense/core#5739]
**Important notices**
Before you add a new report, we ask you kindly to acknowledge the following:
- [ X] I have read the contributing guide lines at https://github.com/opnsense/core/blob/master/CONTRIBUTING.md
- [ X] I am convinced that my issue is new after having checked both open and closed issues at https://github.com/opnsense/core/issues?q=is%3Aissue

**Describe the bug**
A clear and concise description of what the bug is, including last known working version (if any).

.
.

I haven't logged into my system in about a month and I just noticed unfortunately monitoring isn't working again for me.
Pics:
https://i.imgur.com/DxyShfA.png
https://i.imgur.com/q5LI8o7.png
Netflow seems to be enabled and I haven't changed anything.

.
.

*Tip*: to validate your setup was working with the previous version, use opnsense-revert (https://docs.opnsense.org/manual/opnsense_tools.html#opnsense-revert)
**To Reproduce**

N/A appears to be database corruption, I thought ZFS reduced this? I have switched to ZFS a few months ago (fresh install) to avoid problems such as this.

.
.

Reboot the system.
Transfer some traffic (wait 5 minutes)
Click into the insight section to see there's clearly no data.

.
.

**Expected behavior**

Working graphs

.
.

**Describe alternatives you considered**
A clear and concise description of any alternative solutions or workaround you considered.

I've rebooted, confirmed the service is stopping itself.
I did just see "Insight Aggregator" in a stopped state in the dashboard, despite update and reboot. (flowd_aggregate)
However I wouldn't have stopped this. Also starting it, only results in it stopping again.

.
.

**Screenshots**

https://i.imgur.com/DxyShfA.png
https://i.imgur.com/q5LI8o7.png

.
.

**Relevant log files**
If applicable, information from log files supporting your claim.

lowd aggregate died with message Traceback (most recent call last): File "/usr/local/opnsense/scripts/netflow/flowd_aggregate.py", line 160, in run aggregate_flowd(self.config, do_vacuum) File "/usr/local/opnsense/scripts/netflow/flowd_aggregate.py", line 80, in aggregate_flowd stream_agg_object.add(copy.copy(flow_record)) File "/usr/local/opnsense/scripts/netflow/lib/aggregates/interface.py", line 72, in add super(FlowInterfaceTotals, self).add(flow) File "/usr/local/opnsense/scripts/netflow/lib/aggregates/init.py", line 187, in add self._update_cur.execute(self._insert_stmt, flow) sqlite3.DatabaseError: *database disk image is malformed*

.
.

**Additional context**

I previously had a problem, fixed here, with some custom stuff ran on my system, maybe the cause of the issue?
https://github.com/opnsense/core/issues/5579

Filesystem here

root@OPNsense:~ # df
Filesystem 1K-blocks Used Avail Capacity Mounted on
zroot/ROOT/default 21351364 1141180 20210184 5% /
devfs 1 1 0 100% /dev
/dev/ada0p1 266176 1808 264368 1% /boot/efi
zroot/tmp 20210464 280 20210184 0% /tmp
zroot/var/log 20676696 466512 20210184 2% /var/log
zroot 20210280 96 20210184 0% /zroot
zroot/usr/home 20210280 96 20210184 0% /usr/home
zroot/usr/src 20210280 96 20210184 0% /usr/src
zroot/var/audit 20210280 96 20210184 0% /var/audit
zroot/var/crash 20210280 96 20210184 0% /var/crash
zroot/usr/ports 20210280 96 20210184 0% /usr/ports
zroot/var/mail 20210312 128 20210184 0% /var/mail
zroot/var/tmp 20210292 108 20210184 0% /var/tmp
devfs 1 1 0 100% /var/dhcpd/dev
devfs 1 1 0 100% /var/unbound/dev

.
.

**Environment**
Software version used and hardware type if relevant, e.g.:

OPNsense 22.1.6-amd64
FreeBSD 13.0-STABLE
OpenSSL 1.1.1n 15 Mar 2022

.
.

Have I done something wrong? I actually switched to ZFS to get a more resilient filesystem. I was having some serious power failure issues, possibly it was that?

Any advice would be so lovely, thank you.

P.S are there any scripts which email you when a service stops or breaks?
Thanks for the hard work!

P.S Previously logged here:
https://github.com/opnsense/core/issues/5734
but was dropped to low priority.

**Comment by jaxjexjox** [comment]
Hi Team,

I apologise for bumping this, look I'm fine even scapping my entire data history if necessary just to get this working again.
I'd like to think it doesn't break again in future as it's been a little unfortunate.

Please advise a shell command I could drop in to totally zap the database fresh and therefore ensure it starts recording again?

I hope the information I provided above helps.

**Comment by jaxjexjox** [comment]
Ok I have used the menu under settings to reset RRD data
Reset netflow data
and Repair netflow data

I hit all 3 twice, rebooted, I've got reporting back.   

I suspect my corruption was due to power issues in my location where we lost power (a lot) over the course of several weeks.
Though I thought ZFS was more resilient, based on posts here.

So I hope this helps others who google with a similar issue.

Closing job, thanks for the work team.
